src/Link.tsx

```diff
@@ -15,8 +24,9 @@ function Link({
   exact = false,
   onClick,
   target,
+  children,
   ...props
-}) {
+}: any) {
```

Comment:
after a solid 1-2 hrs of trying to make it work with existing types, I gave up. I think those need to be simplified a bit first

Reply:
agree they should be simplified